src/InitModal.tsx

```diff
@@ -71,7 +71,7 @@ export default function InitModal() {
   };
 
   const [projectName, setProjectName] = React.useState('');
-  const handleProjectNameChange = (event) => {
+  const handleProjectNameChange = (event: any) => {
```

Comment:
event type should be `Event`

Reply:
didn't know what to put there, will fix

Reply:
It ended up being `React.ChangeEvent<HTMLInputElement>`